app/controllers/dashboard_controller.rb

```diff
@@ -34,17 +28,11 @@ def dashboard_role
     end
   end
 
-
-  # Public Dashboard - list of all collections
   def index
-    if Collection.all.count > 1000
```

Comment:
This was 6 years ago, not sure if it is still relevant today (I think no, so I removed it)

Reply:
Okay, what this really is about is a differentiator between the SaaS installation on fromthepage.com and the different on-premises installations like @bencomp is running.  For FromThePage.com, users mainly want to navigate different institutions who run projects on the platform, so a list of institutional project owners makes sense.  For on-premises installations, users will be navigating different collections within a single institution.

So on-premises installations want the collection list page, while the SaaS installation wants a find-a-project page.  This logic `Collection.all.count > 1000` is a clunky way of asking "Am I running on FromThePage.com, or am I running on-prem?"  Alternatives might be to just have different code in the `main` branch vs. `development`/`fromthepage.com`.

Thoughts?

Reply:
Okay, the other place where we have different behavior based on installation type is the splash page and footer.  In those cases, we have code in `main` which overrides the logic in `development` to display a different view.

I'm tempted to do the same here, rather than looking at a variable or environment.

What we definitely do _not_ want is a scenario in which someone installing the `main` branch has to do any extra steps (like sent an env var) to avoid seeing our SaaS marketing/pricing pages.  `main` should show the on-prem options--the collection listing, in this case--by default.

Reply:
This seems like it should be its own ticket then. I'm reverting old implementation for now